src/how_to_contribute/how_to_contribute.tex

```diff
@@ -102,11 +102,14 @@ \section{Cloner le répertoire contenant le fork}
 \begin{frame}[fragile]
     \frametitle{Avoir le dossier des synthèses sur son ordinateur}
     \begin{wideitemize}
-        \item Se connecter à \lstinline|git|
+        \item Se connecter à \lstinline|Git|. Sous Windows: clic droit à la racine (dans le dossier \lstinline|Syntheses|) + \lstinline|Git Gash Here|.
```

Comment:
```suggestion
        \item Se connecter à \lstinline|Git|. Sous Windows: clic droit à la racine (dans le dossier \lstinline|Syntheses|) + \lstinline|Git Bash Here|.
```

Reply:
Tu as gashé une bonne blague là

Reply:
En plus je me suis encore demandé si je connaissais cette commande...

Reply:
Tu es sur Mac toi? :smile:  (j'ai mis ça vu pour attirer de nouveaux utilisateurs, qui sont quasi tous sur Windows by the way... Je me dis que celui qui est sur Linux sait déjà comment marche un terminal :smile: )

Reply:
Yep, suis sur macOS.